docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-wide-guide-card/__tests__/cgdp-wide-guide-card.dom.ts

```diff
@@ -0,0 +1,53 @@
+export const cgdpWideGuideCardDom = `
+<div class="usa-section" data-eddl-landing-row="">
```

Comment:
Check this HTML for outdated cgdp-wide-guide-card references. Also it should match the template HTML. The image is being set as a background image and this content has a cgdp-wide-guide-card__image element.